var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -165,10 +165,10 @@ class Conduit(CMakePackage):
     phases = ['hostconfig', 'cmake', 'build', 'install']
 
     def flag_handler(self, name, flags):
-        if name in ('cflags', 'cxxflags', 'fflags'):
-            # the package manages these flags in another way
-            return (None, None, None)
-        return (flags, None, None)
+        if '%cce' in self.spec:
```

Comment:
To follow this package's convention of handling compiler flags, you should do to this down around line 360 or so.